packages/angular/ng-package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "node_modules/ng-packagr/ng-package.schema.json",
+  "dest": "dist",
+  "lib": {
+    "entryFile": "src/index.ts",
+    "umdModuleIds": {
+      "@sentry/browser": "Sentry",
+      "@sentry/utils": "utils"
+    }
+  },
+  "whitelistedNonPeerDependencies": [
```

Comment:
Think this is called `allowedNonPeerDependencies` now? https://github.com/ng-packagr/ng-packagr/blob/master/docs/dependencies.md#allowing-in-the-dependencies-section

Reply:
I wanted to change this property to `allowedNonPeerDependencies` but it was introduced in a later version of `ng-packagr` than the one we're using. I could go ahead and try to upgrade `ng-packagr` but I'm pretty sure that new versions supporting `allowedNonPeerDependencies` will not build APF v10 but some newer version which probably breaks Angular 10 compatibility.